src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1365,6 +1366,16 @@ class ContentExplorer extends Component<Props, State> {
         }
     };
 
+    /**
+     * Returns whether the currently focused element is an item
+     *
+     * @returns {bool}
+     */
+    isFocusedOnItem = () => {
+        const focusedElementClassList = document.activeElement?.classList;
+        return focusedElementClassList && focusedElementClassList.contains('be-item-label');
```

Comment:
I can't do `return document.activeElement?.classList?.contains('be-item-label');` because flow doesnt support